### PR TITLE
[WIP] feat: Add My Apps tab [skip ci]

### PR DESCRIPTION
**Pull request by enguerran**
Please make sure the following boxes are checked:

- [ ] PR is not too big
- [ ] it improves UX & DX in some way
- [ ] it includes tests matching the implementation changes
- [ ] it includes relevant documentation

### CODE REVIEW

File: gui/elm/Window/Tray.elm [hunk 1]
```diff
@@ -29,17 +29,22 @@ import Window.Tray.UserActionRequiredPage as UserActionRequiredPage
...
+
 type Page
     = DashboardPage
     | SettingsPage
+    | MyAppsPage
 
 
 type alias Model =
     { dashboard : Dashboard.Model
+    , settings : Settings.Model
```
Comment (sebn) [inline gui/elm/Window/Tray.elm]: Why did it move up?

File: gui/elm/Window/Tray.elm [hunk 2]
```diff
@@ -48,10 +53,10 @@ type alias Model =
 init : String -> Platform -> Model
 init version platform =
     { dashboard = Dashboard.init
+    , settings = Settings.init version
```
Comment (sebn) [inline gui/elm/Window/Tray.elm]: Why did it move up too?

**Comment by enguerran** [comment]
In order to load a React app like https://github.com/cozy/cozy-home in a tab, I first wrote everything needed to load a JS app (React or else) in a Elm application.

**Comment by enguerran** [comment]
I manage to create a module from the docbase cozy/cozy-home with this webpack configuration
```js
{
  entry: {
    ['cozy-home-for-desktop']: './src/targets/cozy-home-for-desktop/index.jsx'
  },
  output: {
    filename: data =>
      data.chunk.name === 'cozy-home-for-desktop' ? '[name].umd.js' : filename,
    library: 'cozy-home-for-desktop',
    libraryTarget: 'umd'
  },
  module: {
    rules: [
      {
        test: /\.jsx/,
        exclude: /node_modules/,
        loader: "babel-loader",
        options: {
          presets: ["@babel/env"]
        }
      }
    ]
  }
}
```

As you can see in the picture below, a dummy component is loadded from cozy-home-for-desktop in our cozy-labs/cozy-desktop Elm application: 
image

But when I use the bundle config from cozy/cozy-home, with `yarn build`, the bundle does not contain what is needed to display the React component.

**Comment by sebn** [comment]
We'll reopen the PR when we get back to work on this feature.

**Pull request closed without merging**